src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -79,6 +97,32 @@ protected override void BeginProcessing()
         {
             string resolvedpath = string.Empty;
 
+            try
+            {
+                if (Path != null)
+                {
+                    resolvedpath = Context.SessionState.Path.GetUnresolvedProviderPathFromPSPath(Path);
+                    _json = File.ReadAllText(resolvedpath);
+                }
+            }
+            catch (Exception e) when (
+
+                // Handle exceptions related to file access to provide more specific error message
+                // https://docs.microsoft.com/en-us/dotnet/standard/io/handling-io-errors
+                e is IOException ||
+                e is UnauthorizedAccessException ||
+                e is NotSupportedException ||
+                e is SecurityException)
```

Comment:
I think I'll need to include other exception types like `SessionStateException` here.